code/game/machinery/OpTable.dm

```diff
@@ -25,6 +26,13 @@
 			computer.table = src
 			break
 
+/obj/machinery/optable/Destroy()
+	victim = null
+	if(connected_monitor)
+		connected_monitor.update_victim()
+		connected_monitor.update_optable()
```

Comment:
You need to null `connected_monitor`.

Reply:
I realize in retrospect this is kind of jank, but update_optable nulls the connected_optable's monitor by default before starting any new connection (incl null). See vitals_monitor.dm#L105